gogdl/dl/dl_utils.py

```diff
@@ -49,7 +50,9 @@ def get_secure_link(api_handler, path, gameId, generation=2):
     r = api_handler.session.get(url)
     if not r.ok:
         print(f"ERROR getting secure link for {path}")
-        exit(1)
+        time.sleep(0.2)
+        return get_secure_link(api_handler, path, gameId, generation)
```

Comment:
instead of exit we retry, because if we exit then the new secure link flow never ends and all threads are stuck waiting for this thread to finish